CIP-????/README.md

```diff
@@ -0,0 +1,263 @@
+
+The ledger rules currently disallow depositing into account addresses. This restriction will be
+lifted to support ADA deposits. The transaction balancing rules will need to be updated to consider
+these deposits.
+
+**Depositing into an account address does not require a witness from the receiving account.** In
+other words, the receiving staking script does *not* need to be executed. This behavior mirrors how
+UTxOs can be created at script addresses without having to execute the spending script.
+
+> [!IMPORTANT]
+> Adding support for depositing native assets into account addresses is **out of scope** for this
+> CIP. There are open questions for *how* to safely add support for native assets discussed in the
+> [Rationale section](#future-cip---native-asset-deposit-support) of this CIP.
+
+In order to deposit ADA into an account address, the receiving account address *must* be registered.
+
+> [!NOTE]
+> Withdrawing from the account address still requires delegation to a DRep.
+
+When a deposit is made, plutus scripts can see the deposit amount in their contexts similarly to how
+they can see the withdrawal amount. Since it is not recommended to enable negative numbers in the
+[withdrawal
+field](https://github.com/IntersectMBO/cardano-ledger/blob/cd7a8fd7ca833d120b87563a7e99e9b8c1db2dc1/eras/conway/impl/cddl-files/conway.cddl#L136)
+of the transaction body, a new optional field needs to be added to the transaction body:
+
+```cbor
+transaction_body = 
+  {   0  : set<transaction_input>         
+  ,   1  : [* transaction_output]      
+  ...
+  , ? 23 : account_deposits ; new field
+  }
+
+account_deposits = {+ reward_account => coin} ; same definition as current withdrawals
+```
+
+And here is the associated change to the `TxInfo` seen by plutus scripts:
+
+```haskell
+data TxInfo = TxInfo
+  { txInfoInputs                :: [TxInInfo]
+  , txInfoReferenceInputs       :: [TxInInfo]
+  , txInfoOutputs               :: [TxOut]
+  , txInfoFee                   :: Value
+  , txInfoMint                  :: Value
+  , txInfoTxCerts               :: [TxCert]
+  , txInfoWdrl                  :: Map Credential Haskell.Integer
+  ...
+  , txInfoDeposits              :: Map Credential Haskell.Integer -- ^ New field.
+  }
+```
+
+
+> [!NOTE]
+> See the [Rationale section](#future-cip---native-asset-deposit-support) for why `account_deposits`
+> uses `coin` instead of preparing the stage for future native asset support.
+
+### Account Balance Intervals
+
+In order to give plutus scripts some idea of what is happening with the account address without
+introducing a dependency on global state which can cause contention, transactions can specify
+account balance intervals similarly to how time intervals are set. In order for the transaction to
+be valid, the account's actual balance *must* fall within the validity interval. Both of the
+interval bounds are *inclusive*. The account balance intervals will be passed to plutus scripts as
```

Comment:
I think the standard way to define an interval is inclusive lower bound and exclusive upper bound. This allows intervals to be cleanly partitioned without overlap and without having to calculate (i-1) boundary values. For example: [1,10) can be split into [1,5) and [5,10). If you tried the same thing with both-inclusive bounds, you'd have to split [1,9] as [1,4] and [5,9].